clicommand/agent_start.go

```diff
@@ -84,6 +84,7 @@ type AgentStartConfig struct {
 	HealthCheckAddr            string   `cli:"health-check-addr"`
 	MetricsDatadog             bool     `cli:"metrics-datadog"`
 	MetricsDatadogHost         string   `cli:"metrics-datadog-host"`
+	TracingDatadogAddr         string   `cli:"tracing-datadog-addr"`
```

Comment:
Is an addr different from a host, above? It sounds like it might be...? It would be great to use consistent terms with consistent content expectations.

Reply:
It could be but chances are it'll be the same. We could just have one but that'd break backwards compatibility with configurations. Or we reuse the metrics Datadog host config, which then is 'incorrect' when read in the tracing code. But neither of these are strictly terrible issues. Open to either or other ideas.

Reply:
Addr includes a port, where I believe the host shouldn't, though it seems like MetricsDatadogHost does support the port, which is indeed a bit confusing.

Also as far as sharing host between metrics and tracing, it can be different. Most DataDog libraries support specifying these separately, and there are some products that require them to be different, such as Lightstep's DataDog ingest support.

Reply:
I think this is largely resolved with the new approach (no BK-specific configuration env vars - just a backend env var and then use the tracing libs' common config env vars like DD_AGENT_HOST).